From 977c834b292d78bcb4ae548e2f88d3a88e26219d Mon Sep 17 00:00:00 2001
From: SeanMurillo <sean.d.murillo@gmail.com>
Date: Sat, 15 Oct 2022 18:01:00 -0400
Subject: [PATCH] a


diff --git a/proxy/src/main/java/net/md_5/bungee/command/CommandInfiniteBungee.java b/proxy/src/main/java/net/md_5/bungee/command/CommandInfiniteBungee.java
index 5a368d19..d6326c12 100644
--- a/proxy/src/main/java/net/md_5/bungee/command/CommandInfiniteBungee.java
+++ b/proxy/src/main/java/net/md_5/bungee/command/CommandInfiniteBungee.java
@@ -5,7 +5,7 @@ import net.md_5.bungee.api.CommandSender;
 import net.md_5.bungee.api.ProxyServer;
 import net.md_5.bungee.api.plugin.Command;
 
-public class CommandBungee extends Command
+public class CommandInfiniteBungee extends Command
 {
 
     public CommandBungee()
-- 
2.32.1 (Apple Git-133)

