From 5b102da313ab6fafcc2de094cb1c8525cd125aae Mon Sep 17 00:00:00 2001
From: SeanMurillo <sean.d.murillo@gmail.com>
Date: Sat, 15 Oct 2022 18:03:08 -0400
Subject: [PATCH] 2


diff --git a/proxy/src/main/java/net/md_5/bungee/command/CommandInfiniteBungee.java b/proxy/src/main/java/net/md_5/bungee/command/CommandInfiniteBungee.java
index d6326c12..1429163a 100644
--- a/proxy/src/main/java/net/md_5/bungee/command/CommandInfiniteBungee.java
+++ b/proxy/src/main/java/net/md_5/bungee/command/CommandInfiniteBungee.java
@@ -8,7 +8,7 @@ import net.md_5.bungee.api.plugin.Command;
 public class CommandInfiniteBungee extends Command
 {
 
-    public CommandBungee()
+    public CommandInfiniteBungee()
     {
         super( "bungee" );
     }
-- 
2.32.1 (Apple Git-133)

