From c14131478746d8ee1667a280716e8f63e743e5ed Mon Sep 17 00:00:00 2001
From: SeanMurillo <sean.d.murillo@gmail.com>
Date: Tue, 13 Dec 2022 22:51:24 -0500
Subject: [PATCH] AddBugsCommand


diff --git a/proxy/src/main/java/net/md_5/bungee/BungeeCord.java b/proxy/src/main/java/net/md_5/bungee/BungeeCord.java
index f65ae914..3d2505eb 100644
--- a/proxy/src/main/java/net/md_5/bungee/BungeeCord.java
+++ b/proxy/src/main/java/net/md_5/bungee/BungeeCord.java
@@ -81,6 +81,7 @@ import net.md_5.bungee.command.CommandBungee;
 import net.md_5.bungee.command.CommandInfiniteBungee;
 import net.md_5.bungee.command.CommandEnd;
 import net.md_5.bungee.command.CommandIP;
+import net.md_5.bungee.command.CommandBugs;
 import net.md_5.bungee.command.CommandHelp;
 import net.md_5.bungee.command.CommandDiscord;
 import net.md_5.bungee.command.CommandInfinite;
@@ -242,6 +243,7 @@ public class BungeeCord extends ProxyServer
         getPluginManager().registerCommand( null, new CommandEnd() );
         getPluginManager().registerCommand( null, new CommandIP() );
         getPluginManager().registerCommand( null, new CommandHelp() );
+        getPluginManager().registerCommand( null, new CommandBugs() );
         getPluginManager().registerCommand( null, new CommandDiscord() );
         getPluginManager().registerCommand( null, new CommandInfinite() );
         getPluginManager().registerCommand( null, new CommandInfiniteBungee() );
diff --git a/proxy/src/main/java/net/md_5/bungee/command/CommandBugs.java b/proxy/src/main/java/net/md_5/bungee/command/CommandBugs.java
new file mode 100644
index 00000000..5054d4ae
--- /dev/null
+++ b/proxy/src/main/java/net/md_5/bungee/command/CommandBugs.java
@@ -0,0 +1,21 @@
+package net.md_5.bungee.command;
+
+import net.md_5.bungee.api.ChatColor;
+import net.md_5.bungee.api.CommandSender;
+import net.md_5.bungee.api.ProxyServer;
+import net.md_5.bungee.api.plugin.Command;
+
+public class CommandBugs extends Command
+{
+
+    public CommandBugs()
+    {
+        super( "bugs", "infinite.command.bugs", "bug" );
+    }
+
+    @Override
+    public void execute(CommandSender sender, String[] args)
+    {
+        sender.sendMessage( ChatColor.AQUA + "§bGo to our discord server to report bugs! https://discord.gg/mWxUFqsynj " );
+    }
+}
-- 
2.32.1 (Apple Git-133)

