From 967921d06527d1f05896584e396ab6256dca5abf Mon Sep 17 00:00:00 2001
From: SeanMurillo <sean.d.murillo@gmail.com>
Date: Mon, 5 Dec 2022 20:12:32 -0500
Subject: [PATCH] Cat


diff --git a/proxy/src/main/java/net/md_5/bungee/BungeeCord.java b/proxy/src/main/java/net/md_5/bungee/BungeeCord.java
index 563f8ab5..ccf4ad43 100644
--- a/proxy/src/main/java/net/md_5/bungee/BungeeCord.java
+++ b/proxy/src/main/java/net/md_5/bungee/BungeeCord.java
@@ -82,7 +82,7 @@ import net.md_5.bungee.command.CommandIP;
 import net.md_5.bungee.command.CommandHelp;
 import net.md_5.bungee.command.CommandDiscord;
 import net.md_5.bungee.command.CommandInfinite;
-import net.md_5.bungeee.command.CommandCat;
+import net.md_5.bungee.command.CommandCat;
 import net.md_5.bungee.command.CommandPerms;
 import net.md_5.bungee.command.CommandReload;
 import net.md_5.bungee.command.ConsoleCommandSender;
-- 
2.32.1 (Apple Git-133)

