From fe32ae17174ba8c2158e57f16894a131534605e6 Mon Sep 17 00:00:00 2001
From: SeanMurillo <sean.d.murillo@gmail.com>
Date: Sat, 24 Dec 2022 16:06:24 -0500
Subject: [PATCH] SmallTweaks


diff --git a/module/cmd-alert/src/main/java/net/md_5/bungee/module/cmd/alert/CommandAlert.java b/module/cmd-alert/src/main/java/net/md_5/bungee/module/cmd/alert/CommandAlert.java
index 5e24c2c1..80918f7c 100644
--- a/module/cmd-alert/src/main/java/net/md_5/bungee/module/cmd/alert/CommandAlert.java
+++ b/module/cmd-alert/src/main/java/net/md_5/bungee/module/cmd/alert/CommandAlert.java
@@ -11,7 +11,7 @@ public class CommandAlert extends Command
 
     public CommandAlert()
     {
-        super( "alert", "infinitebungee.command.alert" );
+        super( "alert", "infinite.command.alert" );
     }
 
     @Override
diff --git a/module/cmd-find/src/main/java/net/md_5/bungee/module/cmd/find/CommandFind.java b/module/cmd-find/src/main/java/net/md_5/bungee/module/cmd/find/CommandFind.java
index 0f9ba4a2..a4581eab 100644
--- a/module/cmd-find/src/main/java/net/md_5/bungee/module/cmd/find/CommandFind.java
+++ b/module/cmd-find/src/main/java/net/md_5/bungee/module/cmd/find/CommandFind.java
@@ -10,7 +10,7 @@ public class CommandFind extends PlayerCommand
 
     public CommandFind()
     {
-        super( "find", "infinitebungee.command.find" );
+        super( "find", "infinite.command.find" );
     }
 
     @Override
diff --git a/module/cmd-kick/src/main/java/net/md_5/bungee/module/cmd/kick/CommandKick.java b/module/cmd-kick/src/main/java/net/md_5/bungee/module/cmd/kick/CommandKick.java
index e5852a95..bd836b2c 100644
--- a/module/cmd-kick/src/main/java/net/md_5/bungee/module/cmd/kick/CommandKick.java
+++ b/module/cmd-kick/src/main/java/net/md_5/bungee/module/cmd/kick/CommandKick.java
@@ -18,7 +18,7 @@ public class CommandKick extends Command implements TabExecutor
 
     public CommandKick()
     {
-        super( "gkick", "infinitebungee.command.kick" );
+        super( "gkick", "infinite.command.kick" );
     }
 
     @Override
diff --git a/module/cmd-list/src/main/java/net/md_5/bungee/module/cmd/list/CommandList.java b/module/cmd-list/src/main/java/net/md_5/bungee/module/cmd/list/CommandList.java
index 69d453ba..862cb05c 100644
--- a/module/cmd-list/src/main/java/net/md_5/bungee/module/cmd/list/CommandList.java
+++ b/module/cmd-list/src/main/java/net/md_5/bungee/module/cmd/list/CommandList.java
@@ -19,7 +19,7 @@ public class CommandList extends Command
 
     public CommandList()
     {
-        super( "glist", "infinitebungee.command.list" );
+        super( "glist", "infinite.command.list" );
     }
 
     @Override
diff --git a/module/cmd-send/src/main/java/net/md_5/bungee/module/cmd/send/CommandSend.java b/module/cmd-send/src/main/java/net/md_5/bungee/module/cmd/send/CommandSend.java
index eff20de1..b35ff3ec 100644
--- a/module/cmd-send/src/main/java/net/md_5/bungee/module/cmd/send/CommandSend.java
+++ b/module/cmd-send/src/main/java/net/md_5/bungee/module/cmd/send/CommandSend.java
@@ -93,7 +93,7 @@ public class CommandSend extends Command implements TabExecutor
 
     public CommandSend()
     {
-        super( "send", "infinitebungee.command.send" );
+        super( "send", "infinite.command.send" );
     }
 
     @Override
diff --git a/module/cmd-server/src/main/java/net/md_5/bungee/module/cmd/server/CommandServer.java b/module/cmd-server/src/main/java/net/md_5/bungee/module/cmd/server/CommandServer.java
index a5b8a4c9..163997c0 100644
--- a/module/cmd-server/src/main/java/net/md_5/bungee/module/cmd/server/CommandServer.java
+++ b/module/cmd-server/src/main/java/net/md_5/bungee/module/cmd/server/CommandServer.java
@@ -26,7 +26,7 @@ public class CommandServer extends Command implements TabExecutor
 
     public CommandServer()
     {
-        super( "server", "infinitebungee.command.server" );
+        super( "server", "infinite.command.server" );
     }
 
     @Override
diff --git a/proxy/src/main/java/net/md_5/bungee/command/CommandEnd.java b/proxy/src/main/java/net/md_5/bungee/command/CommandEnd.java
index 34075ff8..1205e608 100644
--- a/proxy/src/main/java/net/md_5/bungee/command/CommandEnd.java
+++ b/proxy/src/main/java/net/md_5/bungee/command/CommandEnd.java
@@ -15,7 +15,7 @@ public class CommandEnd extends Command
 
     public CommandEnd()
     {
-        super( "end", "infinitebungee.command.end" );
+        super( "end", "infinite.command.end" );
     }
 
     @Override
diff --git a/proxy/src/main/java/net/md_5/bungee/command/CommandIP.java b/proxy/src/main/java/net/md_5/bungee/command/CommandIP.java
index 2f706f53..b4518347 100644
--- a/proxy/src/main/java/net/md_5/bungee/command/CommandIP.java
+++ b/proxy/src/main/java/net/md_5/bungee/command/CommandIP.java
@@ -9,7 +9,7 @@ public class CommandIP extends PlayerCommand
 
     public CommandIP()
     {
-        super( "ip", "infinitebungee.command.ip" );
+        super( "ip", "infinite.command.ip" );
     }
 
     @Override
diff --git a/proxy/src/main/java/net/md_5/bungee/command/CommandInfiniteBungee.java b/proxy/src/main/java/net/md_5/bungee/command/CommandInfiniteBungee.java
index 9c53c483..505937bc 100644
--- a/proxy/src/main/java/net/md_5/bungee/command/CommandInfiniteBungee.java
+++ b/proxy/src/main/java/net/md_5/bungee/command/CommandInfiniteBungee.java
@@ -10,12 +10,12 @@ public class CommandInfiniteBungee extends Command
 
     public CommandInfiniteBungee()
     {
-        super( "infinitebungee" );
+        super( "proxyserver" );
     }
 
     @Override
     public void execute(CommandSender sender, String[] args)
     {
-        sender.sendMessage( ChatColor.BLUE + "This server is running InfiniteBungee by the Infinite Development Team, a fork of Waterfall by PaperMC and md_5" );
+        sender.sendMessage( ChatColor.BLUE + "This server is running InfinieBungee version " + ProxyServer.getInstance().getVersion() + " by the Infinite Development Team, a fork of Waterfall by PaperMC and md_5" );
     }
 }
diff --git a/proxy/src/main/java/net/md_5/bungee/command/CommandPerms.java b/proxy/src/main/java/net/md_5/bungee/command/CommandPerms.java
index fff924e8..18422669 100644
--- a/proxy/src/main/java/net/md_5/bungee/command/CommandPerms.java
+++ b/proxy/src/main/java/net/md_5/bungee/command/CommandPerms.java
@@ -12,7 +12,7 @@ public class CommandPerms extends Command
 
     public CommandPerms()
     {
-        super( "infiniteperms" );
+        super( "perms" );
     }
 
     @Override
diff --git a/proxy/src/main/java/net/md_5/bungee/command/CommandReload.java b/proxy/src/main/java/net/md_5/bungee/command/CommandReload.java
index 0af18cc6..ebb0dfce 100644
--- a/proxy/src/main/java/net/md_5/bungee/command/CommandReload.java
+++ b/proxy/src/main/java/net/md_5/bungee/command/CommandReload.java
@@ -11,7 +11,7 @@ public class CommandReload extends Command
 
     public CommandReload()
     {
-        super( "greload", "infinitebungee.command.reload" );
+        super( "greload", "infinite.command.reload" );
     }
 
     @Override
diff --git a/proxy/src/main/resources/messages.properties b/proxy/src/main/resources/messages.properties
index ae60b896..de0c7989 100644
--- a/proxy/src/main/resources/messages.properties
+++ b/proxy/src/main/resources/messages.properties
@@ -1,24 +1,24 @@
-alert=\u00a76[\u00a7e!\u00a76]\u00a7r
+alert=\u00a76[\u00a7e!\u00a76] \u00a7r
 already_connected=\u00a7cYou are already connected to this server!
-already_connected_proxy=\u00a7aYou are already connected to \u00a7bInfinite\u00a7a!
+already_connected_proxy=\u00a7aYou are already connected to \u00a7bInfinite Network\u00a7a!
 already_connecting=\u00a7cAlready connecting to this server!
-command_list=\u00a7eServer:\u00a73 {0} \u00a7aOnline: \u00a76{1} \u00a73People: \u00a7b{2}
+command_list=\u00a7eServer:\u00a73 {0} \u00a7aOnline: \u00a76{1} \u00a73Players: \u00a7b{2}
 connect_kick=Could not connect you to the target server: {1}
 current_server=You are connected to server {0}
 fallback_kick=Could not connect you to the target server: Connection refused.
 fallback_lobby=Could not connect you to the target server, you have been moved to a fallback server.
 lost_connection=[Proxy] Lost connection to server.
 mojang_fail=Error occurred while contacting login servers, are they down?
-no_permission=\u00a7cYou don't have permission to use that command!
+no_permission=\u00a7cYou don''t have permission to use that command!
 no_server=\u00a7cThat server does not exist!
-no_server_permission=\u00a7cYou don't have permission to access this custom server.
-outdated_client=\u00a7cOutdated client! \u00a7aTo play on \u00a7bInfinite\u00a7a please use {0}!
+no_server_permission=\u00a7cYou don''t have permission to access this custom server.
+outdated_client=\u00a7cOutdated client! \u00a7aTo play on \u00a7bInfinite Network\u00a7a please use 1.8+!
 outdated_server=Outdated server! Please let us know of this error on our discord!
 proxy_full=\u00a7bInfinite\u00a7a is full, check back later!
-restart=\u00a7bInfinite\u00a7a is restarting!
+restart=\u00a7bInfinite Network\u00a7a is restarting!
 server_list=\u00a7aYou may connect to these servers right now:
 server_went_down=The server you were previously on went down, you have been connected to a fallback server
-total_players=\u00a7aTotal players on \u00a7bInfinite\u00a7a:\u00a76 {0}
+total_players=\u00a7aTotal players on \u00a7bInfinite Network\u00a7a:\u00a76 {0}
 name_invalid=Username contains invalid characters.
 ping_cannot_connect=Can''t connect to server.
 offline_mode_player=Not authenticated with Minecraft.net
@@ -36,8 +36,8 @@ user_online_at=\u00a7b{0} \u00a7ais online at {1}
 send_cmd_usage=\u00a7aNot enough arguments, usage: /send [server|player|all|current] [target]
 player_only=\u00a7cOnly in game players can use this command!
 you_got_summoned=\u00a7b\u00a7l» \u00a77Connecting to {0}..
-command_perms_groups=\u00a7aYou have this group: {0}
+command_perms_groups=\u00a7aYou are on group: {0}
 command_perms_permission=\u00a7b- {0}
 command_ip=\u00a73IP of {0} is {1}
 illegal_chat_characters=\u00a7cillegal characters in chat
-kick_message=\u00a7cYou have been kicked.
+kick_message=\u00a7cYou have been kicked from \u00a7bInfinite Network\u00a7c!
-- 
2.32.1 (Apple Git-133)

