From bb2fd3db681ef6e258d6304aab1dbde7ae08e5f9 Mon Sep 17 00:00:00 2001
From: SeanMurillo <sean.d.murillo@gmail.com>
Date: Tue, 17 Jan 2023 21:44:29 -0500
Subject: [PATCH] Add/s


diff --git a/module/cmd-server/src/main/java/net/md_5/bungee/module/cmd/server/CommandServer.java b/module/cmd-server/src/main/java/net/md_5/bungee/module/cmd/server/CommandServer.java
index 163997c0..6403a2ee 100644
--- a/module/cmd-server/src/main/java/net/md_5/bungee/module/cmd/server/CommandServer.java
+++ b/module/cmd-server/src/main/java/net/md_5/bungee/module/cmd/server/CommandServer.java
@@ -26,7 +26,7 @@ public class CommandServer extends Command implements TabExecutor
 
     public CommandServer()
     {
-        super( "server", "infinite.command.server" );
+        super( "server", "s", "connect", "infinite.command.server" );
     }
 
     @Override
diff --git a/proxy/src/main/java/net/md_5/bungee/BungeeCord.java b/proxy/src/main/java/net/md_5/bungee/BungeeCord.java
index 3d2505eb..65055396 100644
--- a/proxy/src/main/java/net/md_5/bungee/BungeeCord.java
+++ b/proxy/src/main/java/net/md_5/bungee/BungeeCord.java
@@ -518,7 +518,7 @@ public class BungeeCord extends ProxyServer
                     } catch (InterruptedException ignored) {}
         }
 
-        getLogger().info( "Thank you and goodbye" );
+        getLogger().info( "InfiniteBungee successfully stoped" );
         // Need to close loggers after last message!
         org.apache.logging.log4j.LogManager.shutdown(); // Waterfall
 
-- 
2.32.1 (Apple Git-133)

