From 3ae3ba32bed4cfe6797d65b79648b396ebefa51e Mon Sep 17 00:00:00 2001
From: SeanMurillo <sean.d.murillo@gmail.com>
Date: Fri, 10 Feb 2023 09:06:53 -0500
Subject: [PATCH] FixRestrictedServerPerm2--10-23


diff --git a/proxy/src/main/java/net/md_5/bungee/BungeeServerInfo.java b/proxy/src/main/java/net/md_5/bungee/BungeeServerInfo.java
index 1ea654f1..b9fed5b8 100644
--- a/proxy/src/main/java/net/md_5/bungee/BungeeServerInfo.java
+++ b/proxy/src/main/java/net/md_5/bungee/BungeeServerInfo.java
@@ -75,7 +75,7 @@ public class BungeeServerInfo implements ServerInfo
     @Override
     public String getPermission()
     {
-        return "bungeecord.server." + name;
+        return "infinite.server." + name;
     }
 
     @Override
-- 
2.37.1 (Apple Git-137.1)

